### PR TITLE
[BACKEND-616] Defaulting buildV9Directly to true

### PR DESCRIPTION
**Pull request by gkc2104**
[BACKEND-616] FIxed formatting

### CODE REVIEW

**Review by drcrallen — APPROVED** [review]

File: core/src/main/scala/com/metamx/tranquility/druid/DruidTuning.scala [hunk 1]
```diff
@@ -47,16 +47,22 @@ object DruidTuning
     *
     * @param maxRowsInMemory           number of rows to aggregate before persisting
     * @param intermediatePersistPeriod period that determines the rate at which intermediate persists occur
+<<<<<<< HEAD
```
Comment (isimonenko) [inline core/src/main/scala/com/metamx/tranquility/druid/DruidTuning.scala]: Is it a merge conflict artifact?

**Review by isimonenko — CHANGES_REQUESTED** [review]

File: core/src/main/scala/com/metamx/tranquility/druid/DruidTuning.scala (hunk 1 above)
Comment (drcrallen) [reply core/src/main/scala/com/metamx/tranquility/druid/DruidTuning.scala]: ha! yeah, somehow that one got missed

**Comment by gkc2104** [comment]
@isimonenko thanks for pointing that out, resolved it

File: core/src/main/scala/com/metamx/tranquility/druid/DruidTuning.scala (hunk 1 above)
Comment (logarithm) [reply core/src/main/scala/com/metamx/tranquility/druid/DruidTuning.scala]: We have to have automatic travis tests for all our projects to catch situations like this automatically

**Review by isimonenko — APPROVED** [review]

**Pull request merged**